WebContent/WEB-INF/view/addAndEditBook/index.jsp

```diff
@@ -1,25 +1,26 @@
 <%@ page import="servletTutorial.entities.Book"%>
+<%@ page import="java.util.List"%>
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
 <!DOCTYPE html>
 <html>
 <head>
 	<title>The book - Add book</title>
-	<link rel="icon" href="https://img.icons8.com/cute-clipart/344/book.png" type="image/gif" sizes="16x16">
+	<link rel="icon" href="./static/image/book.png" type="image/gif" sizes="16x16">
 	<link href="./static/css/main.css" rel="stylesheet" type="text/css">
 	<link href="https://fonts.googleapis.com/css?family=Google+Sans:400,500,700,900|Google+Sans+Display:400,500" rel="stylesheet" nonce="fSXOtZ5Ye0RE8N7b_mEygA">
 </head>
 <body>
 	<%
 		Book book = (Book) request.getAttribute("data");
-		String error = (String) request.getAttribute("error");
+		String errors = (String) request.getAttribute("errors");
```

Comment:
Please naming `error` if you have only one error. We use `errors` in case we have a list of errors